File: weeks/01/5_divisible_sum_pairs/divisible_sum_pairs.js
function divisibleSumPairs(n, k, ar) {
    let validPairs = 0;
    
    for (let i = 0; i < n-1; i++) {
        for (let j = i+1; j < n; j++) {
            // Is divisible
            if ((ar[i] + ar[j]) % k === 0) {
                validPairs++;
            }
        }
    }
    
    return validPairs;
}

File: weeks/01/5_divisible_sum_pairs/divisible_sum_pairs.rb
def divisibleSumPairs(n, k, ar)
    valid_pairs = 0
    (0...n-1).each do |i|
        (i+1...n).each do |j|
            # Is divisible
            if (ar[i] + ar[j]) % k == 0
                valid_pairs += 1
            end
        end
    end
    valid_pairs
end